neo/Consensus/ConsensusService.cs

```diff
@@ -151,14 +155,33 @@ private void Log(string message, LogLevel level = LogLevel.Info)
             Plugin.Log(nameof(ConsensusService), level, message);
         }
 
+        private void SendRecoveryMessageIfNecessary()
```

Comment:
@jsolman, the name of the function has `IfNecessary`, but I did not see this IfNecessary check.

Reply:
Go for it. :dagger: